src/ci/init_repo.sh

```diff
@@ -41,8 +41,10 @@ if [ ! -f "$cache_valid_file" ]; then
     rm -rf "$CACHE_DIR"
     mkdir "$CACHE_DIR"
 else
+    set +o errexit
     stat_lines=$(cd "$cache_src_dir" && git status --porcelain | wc -l)
     stat_ec=$(cd "$cache_src_dir" && git status >/dev/null 2>&1 && echo $?)
```

Comment:
It's not part of this PR, but the `&& echo $?` seems a bit suspicious to me -- since `&&` is used `$?` should always be `0` here, right?